README.md

```diff
@@ -8,16 +51,238 @@ so don't try to use it for anything close to a production environment.
+
+… REDACTED …
+Update dependencies
+hack/update_deps.sh
+hack/install_test_deps.sh
+Downloading golangci-lint
+… REDACTED …
+/home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kustomize build config/default | /home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kubectl apply -f -
+namespace/oran-o2ims created
+serviceaccount/oran-o2ims-controller-manager created
+role.rbac.authorization.k8s.io/oran-o2ims-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-manager-role created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-proxy-role created
+rolebinding.rbac.authorization.k8s.io/oran-o2ims-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/oran-o2ims-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/oran-o2ims-proxy-rolebinding created
+configmap/oran-o2ims-env-config created
+service/oran-o2ims-controller-manager-metrics-service created
+deployment.apps/oran-o2ims-controller-manager created
+```
+
+The operator is installed in the `oran-o2ims` namespace, which is created during the install.
+
+```bash
+$ oc get pods -n oran-o2ims-system
+NAME                                             READY   STATUS    RESTARTS   AGE
+oran-o2ims-controller-manager-8668794cdc-27xvf   2/2     Running   0          11s
+```
+
+The operator by default creates a default inventory in the oran-o2ims namespace:
+
+```bash
+$ oc get inventory -n oran-o2ims
+NAME      AGE
+default   4m20s
+```
+
+> :warning: Currently, the following components are enabled by default. Notice as well, that they won't start until the Inventory CR is fully populated.
+
+```sh
+$ oc get inventory -n oran-o2ims -oyaml
+```
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  creationTimestamp: "2024-11-07T09:26:12Z"
+  generation: 1
+  name: default
+  namespace: oran-o2ims
+  resourceVersion: "279507769"
+  uid: aeb85580-f7ca-4861-8c38-3f57902b1617
+spec:
+  alarmSubscriptionServerConfig:
+    enabled: false
+  deploymentManagerServerConfig:
+    backendType: regular-hub
+    enabled: true
+  metadataServerConfig:
+    enabled: true
+  resourceServerConfig:
+    enabled: true
+```
+
+### <a name='Run'></a>Run
+
+The `Inventory` custom resource is the way to configure the different components of the IMS operator. By default the installation of the operator creates an Inventory CRD which is missing mandatory information. To enable the multiple components of the operator we have to patch the Inventory CR named `default`:
+
+```yaml
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/instance: orano2ims-sample
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: default
+  namespace: oran-o2ims
+spec:
```

Comment:
I would add an example of the SMO attributes ... perhaps without TLS and OAuth but at least enough to show how to configure the endpoint/url.

Reply:
Added:

``ỳaml
  smo:
    url: http://10.6.85.1:1080
    registrationEndpoint: /mock_smo/v1/ocloud_observer
```